lib/mri/neo4j/driver/types/uuid.rb

```
@@ -0,0 +1,29 @@
+      # hyphenated form via `to_s`. `new` is private so flavor-agnostic
+      # code goes through `from_string`, the one constructor that also
+      # exists on java.util.UUID.
+      class UUID
+        def self.from_string(value) = new(value)
+        private_class_method :new
+
+        def initialize(value)
+          @value = value.to_s
+        end
+
+        def to_s = @value
```

RuboCop (1.88.2)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)